packages/frontend/src/translations/en.global.json

```diff
@@ -912,6 +912,24 @@
         "loadMore": "Load more"
     },
 
+    "NFTDetail": {
+        "owner": "OWNER",
+        "transfer": "Transfer"
+    },
+
+    "NFTTransfer": {
+      "transfer-nft": "Transfer NFT",
```

Comment:
Can we please keep translation IDs camel cased for consistency sake; `enter-receipt` --> `enterReceipt`, etc.